<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="16" column="4">
       <widget class="QCheckBox" name="checkBox_send_newline">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>发送新行</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="turn_left">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="down">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>↓</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="go_back">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="turn_right">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>d</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="up">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>↑</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="stop">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item row="14" column="7">
       <widget class="QLineEdit" name="sendBox_7"/>
      </item>
      <item row="16" column="5">
       <widget class="QPushButton" name="pushButton_send">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>发送(enter)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="go_forward">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>w</string>
        </property>
       </widget>
      </item>
      <item row="15" column="7">
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item row="10" column="8">
       <widget class="QPushButton" name="sendButton_3">
        <property name="text">
         <string>发送（3）</string>
        </property>
       </widget>
      </item>
      <item row="9" column="8">
       <widget class="QPushButton" name="sendButton_2">
        <property name="text">
         <string>发送（2）</string>
        </property>
       </widget>
      </item>
      <item row="13" column="8">
       <widget class="QPushButton" name="sendButton_6">
        <property name="text">
         <string>发送（6）</string>
        </property>
       </widget>
      </item>
      <item row="14" column="8">
       <widget class="QPushButton" name="sendButton_7">
        <property name="text">
         <string>发送（7）</string>
        </property>
       </widget>
      </item>
      <item row="8" column="7">
       <widget class="QLineEdit" name="sendBox_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="10" column="7">
       <widget class="QLineEdit" name="sendBox_3"/>
      </item>
      <item row="12" column="8">
       <widget class="QPushButton" name="sendButton_5">
        <property name="text">
         <string>发送（5）</string>
        </property>
       </widget>
      </item>
      <item row="11" column="8">
       <widget class="QPushButton" name="sendButton_4">
        <property name="text">
         <string>发送（4）</string>
        </property>
       </widget>
      </item>
      <item row="9" column="7">
       <widget class="QLineEdit" name="sendBox_2"/>
      </item>
      <item row="12" column="7">
       <widget class="QLineEdit" name="sendBox_5"/>
      </item>
      <item row="13" column="7">
       <widget class="QLineEdit" name="sendBox_6"/>
      </item>
      <item row="11" column="7">
       <widget class="QLineEdit" name="sendBox_4"/>
      </item>
      <item row="8" column="8">
       <widget class="QPushButton" name="sendButton_1">
        <property name="text">
         <string>发送（1）</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" rowspan="9" colspan="7">
       <widget class="QTextEdit" name="textEdit_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="7">
       <widget class="QLineEdit" name="lineEdit_send">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="comboBox_baudrate_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_open">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>打开串口(space)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_clear">
        <property name="text">
         <string>清空缓冲区(alt)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>串口助手</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
